millipede/millipede.go

```diff
@@ -7,31 +7,51 @@
 package millipede
 
 import (
-	"strings"
+	"fmt"
 )
 
-type Millipede struct {
-	// Size is the amount of feet pairs
-	Size uint64
+type Pede string
+type Millipede []Pede
```

Comment:
We should keep the Millipede as a structure, so we can add new options easily: `Reverse=true`, `Template="bocal/ascii/"`, etc